src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -208,29 +210,41 @@ def load_relative_risk_data(builder, risk: RiskString, target: TargetString):
     distribution_type = get_distribution_type(builder, risk)
     relative_risk_source = builder.configuration[f'effect_of_{risk.name}_on_{target.name}'][target.measure]
 
-    if relative_risk_source == 'data':
+    if relative_risk_source['relative_risk'] is not None:
+        relative_risk_data = _make_relative_risk_data(builder, float(relative_risk_source['relative_risk']))
+    elif relative_risk_source['log_mean'] is not None:  # log distribution
```

Comment:
should we do some checking to make sure someone doesn't specify a mean and a log_mean and get a possibly unexpected result?